### DISCUSSION TITLE
App no longer works after 3g shutoff by BMW

### DISCUSSION THREAD

**Discussion by nimda5** [BimmerGestalt/AAIdrive#475] [Q&A] [answered]
BMW have posted that they will discontinue connected drive support  after Feb 2022.
More info found on USA: [https://www.bmwusa.com/content/dam/bmwusa/connected-drive/pdf/3G_FAQ.pdf](https://www.bmwusa.com/content/dam/bmwusa/connected-drive/pdf/3G_FAQ.pdf)
How that will affect this great software and what functionality we will still have?
Still 10x for great job.

**Comment by hufman** [comment]
I hope it will continue to work! The protocol itself doesn't use the 3g connection, but I suspect the car uses it to check the subscription status to enable the protocol. You might need to use coding to force-enable the BMW A4A Apps protocol after the subscription service becomes unavailable.

**Reply by mbonus** [reply]
> I hope it will continue to work! The protocol itself doesn't use the 3g connection, but I suspect the car uses it to check the subscription status to enable the protocol. You might need to use coding to force-enable the BMW A4A Apps protocol after the subscription service becomes unavailable.

Well I'm willing to test coding through esys. It currently definitely does not work with my 2015 f30 with nbt id4 (3g).

**Comment by TBGR** [comment, marked as answer]
It should work. I have an NBT retrofitted into a BMW E60, that means I can't pay for any of the subscriptions. As long as "BMW APPS" are coded you should be good.

**Comment by sicktim** [comment]
I am coming from a crosspost. I had BMW Apps previously, but due to this issue they no longer show up. It seems like some coding will be required to activate the BMW Apps again (even if you had the technology package). It doesn't look as simple as a bimmercode session, but rather a more in-depth coding from bimmer-tech.

**Reply by TBGR** [reply]
From what I know if the car did not come with BMW Apps as an option, but you have a connected drive subscription you can access apps while it is active. If it has apps you can access them independently. I think you still need an FSC code to activate them if you code them.

**Comment by E46Fanatic** [comment]
it should be noted that, as far as I know, the support for ConnectedDrive itself will not be discontinued. The problem is, that ConnectedDrive will not work in cars with only 3G sim cards (and not LTE) due to discontinuation of 3g by the operators, so it just wont be able to connect to the network. 

In other words, the ConnectedDrive will not be affected for example in G30 series, which all have LTE cards, not 3G cards. 
At least this is how I understand it.

**Reply by mbonus** [reply]
Can confirm that it no longer works on my 3G 2015 f30. If anybody knows what needs to be coded, to bypass the 3G check and add app connection assistant, please post it

**Reply by LACNR** [reply]
ConnectedDrive Services, BMW Online, TeleServices and Emergency Call continue to work if your country supports 2G Network, which is the old standard (emergency) network. If your vehicle is equipped with a UMTS (3G) Sim it automatically falls to the 2G Network, which is slower but couple of features will still work (BMW Online, Weather, News, E-Mail and Information).

If you previously have purchased Connected Drive it comes with ConnectedDrive Services for lifetime (until 2020 CDS was specifically mentioned to be included when buying a BMW Online Service, it isn't specifically mentioned anymore but remains in active service lists and the description is still available in the store. Cars with BMW Apps as standard will be able to use BMW Apps regardless, however it is possible that a one-time activation of CDS (which comes with a CD Purchase) was required if a included subscribtion wasn't ordered at configuration already. BMW Apps will remain then for lifetime, in case your car has the option of BMW Apps by VIN.

**Comment by hufman** [comment]
Check out [this forum thread](https://f30.bimmerpost.com/forums/showthread.php?t=1030394), where someone is able to provide FSC codes to enable BMW Apps in older cars.
